Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1123,6 +1134,37 @@ void MenuBar::UpdateToolsMenu(const Core::State state)
   }
 }
 
+#ifdef RC_CLIENT_SUPPORTS_RAINTEGRATION
+void MenuBar::UpdateToolsAchievements()
+{
+  auto* dev_menu = AchievementManager::GetInstance().GetDevelopmentMenu();
+  if (dev_menu)
+  {
+    m_achievements_dev_menu->menuAction()->setVisible(true);
+    m_achievements_dev_menu->clear();
+    for (u32 ix = 0; ix < dev_menu->num_items; ix++)
+    {
+      const auto& menu_item = dev_menu->items[ix];
+      if (menu_item.label == nullptr)
+      {
+        m_achievements_dev_menu->addSeparator();
+        continue;
+      }
+      auto* ra_dev_menu_item = m_achievements_dev_menu->addAction(
+          QString::fromLocal8Bit(menu_item.label), this,
+          [menu_item]() { AchievementManager::GetInstance().ActivateDevMenuItem(menu_item.id); });
+      ra_dev_menu_item->setEnabled(menu_item.enabled);
+      ra_dev_menu_item->setCheckable(menu_item.checked);
```

Comment:
Are you sure this line is correct? Does the library not allow for items that are checkable but start out unchecked?